src/crab_verifier.cpp

```diff
@@ -199,7 +198,7 @@ ebpf_analyze_program_for_test(std::ostream& os, const InstructionSeq& prog, cons
     cfg_t cfg = prepare_cfg(prog, info, !no_simplify, false);
     auto [pre_invariants, post_invariants] = crab::run_forward_analyzer(cfg, entry_inv, check_termination);
     checks_db report = generate_report(cfg, pre_invariants, post_invariants);
-    print_report(os, report, prog);
+    print_report(os, report, prog, [](std::ostream& out, const btf_line_info_t& info) { out << info; });
```

Comment:
This lambda is also defined below. I think it's cleaner to have both as static functions outside.

But why use a formatter if the only decision is whether or not to print? Again it feels like I'm missing something. Are you planning to add more line-info formatting?